Session/Meta/AppDelegate.swift

```diff
@@ -133,6 +133,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
                     }
                 )
                 
+                /// Do a little migration on an educated guess of if the user has been asked for local network permission based on calls permission
+                dependencies[defaults: .standard, key: .hasRequestedLocalNetworkPermission] = dependencies[singleton: .storage, key: .areCallsEnabled]
```

Comment:
Should this be an actual migration? Or should it be checking that `hasRequestedLocalNetworkPermission` hasn't been set before we set it's value to the `areCallsEnabled` value? (otherwise could this be overriding the `hasRequestedLocalNetworkPermission` incorrectly if the user had switched it off?)

Reply:
This is a bit tricky. The idea of adding this is just to prevent new users who only installed Session after this update being asked for local network access on first startup. Because the only way we can check the status of local network permission will trigger the system prompt to ask for the permission. It won't affect anything after the user has enabled the calls permission and gone through the permission chain after that.

Reply:
Wait, on a second thought, this might be something useless. Let me think about it again and do some test.

Reply:
Ah ok cool, so it _does_ go into the incorrect state in `didFinishLaunchingWithOptions` but the code in `applicationDidBecomeActive` updates it to be back in the correct state, no worries - thanks for updating the comment 🙏